Library/Formula/pdfgrep.rb

```diff
@@ -15,8 +15,16 @@ class Pdfgrep < Formula
   depends_on "pkg-config" => :build
   depends_on "poppler"
 
+  if build.head?
+    depends_on :automake
+    depends_on :autoconf
+    depends_on "asciidoc" => :build
+  end
```

Comment:
The symbol for is no longer used; use `depends_on "automake" => :build` and `depends_on "autoconf" => :build` instead.

Also, these should be in a `head` block (with the URL) rather than a conditional.